### PR TITLE
feat(ui): Discover table and various other tweaks

### PR DESCRIPTION
**Pull request by ckj**
screen shot 2018-09-06 at 1 16 49 pm

This PR:

- Adds a Results heading
- Tweaks the table styles (they still need a little love)
- Wraps charts in Panels
- Fixes the sidebar overflow issue

### CODE REVIEW

**Comment by ckj** [comment]
Oh, I meant to address the query stats summary but i'm not sure what to do with it tbh

**Comment by lynnagara** [comment]
how about some space below the stats? (to match the one above)

File: src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx [hunk 1]
```diff
@@ -107,59 +109,76 @@ export default class Result extends React.Component {
 
     return (
       <div>
-        {this.renderToggle()}
+        <Flex align="center" mb={space(2)}>
+          <Box flex="1">
+            <Heading>Result</Heading>
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx]: nit: please translate :)

**Review by lynnagara — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx [hunk 2]
```diff
@@ -107,59 +109,76 @@ export default class Result extends React.Component {
...
-            series={basicChartData}
-            height={300}
-            options={{
-              tooltip: Tooltip({
-                formatter: formatTooltip,
-              }),
-            }}
-          />
+          <Panel>
+            <Box px={space(2)} py={space(1.5)}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx]: Can we change this into a styled component? It's re-used multiple times here and also we should limit the use of these presentational props -- it makes things hard to read.

File: src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx [hunk 3]
```diff
@@ -107,59 +109,76 @@ export default class Result extends React.Component {
...
-          <LineChart
-            series={basicChartData}
-            height={300}
-            options={{
-              tooltip: Tooltip({
-                formatter: formatTooltip,
-              }),
-            }}
-          />
+          <Panel>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx]: There's `app/components/charts/panelChart` that we're using for health if we wanted a similar panel.

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx [hunk 4]
```diff
@@ -146,20 +147,16 @@ export default class ResultTable extends React.Component {
           {({width, height}) => (
             <MultiGrid
               ref={ref => (this.grid = ref)}
-              width={width}
+              width={width - 1} // Fix weird 1px right overflow
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: This is not a legal comment

File: src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx (hunk 2 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx]: 👍

File: src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx (hunk 3 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx]: Good idea! I'll use that instead.

**Review by ayesha-omarali — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx [hunk 5]
```diff
@@ -240,9 +240,9 @@ export default class OrganizationDiscover extends React.Component {
         </Header>
         <DiscoverBody>
           <Sidebar w={320}>
-            <SidebarHeader align="center" px={space(4)} py={space(1.5)}>
+            <SidebarHeader align="center" px={space(4)} py={space(2)}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx]: Can we move padding into the styled component?

**Review by billyvg — APPROVED** [review]

**Pull request merged**